packages-new/components/range/src/components/osds-range/osds-range.spec.ts

```diff
@@ -25,10 +28,23 @@ describe('spec:osds-range', () => {
   let instance: OsdsRange;
   let htmlInput: HTMLInputElement | null | undefined;
   let controller: OdsRangeController;
-
-  async function setup({ attributes = {} }: { attributes?: Partial<OdsRangeAttributes> } = {}) {
-    const minimalAttributes: OdsRangeAttributes = OdsCreateAttributes(attributes, odsRangeBaseAttributes);
-    const stringAttributes = OdsComponentAttributes2StringAttributes<OdsRangeAttributes>(minimalAttributes, odsRangeDefaultAttributes);
+  const baseAttribute = {
+    color: OdsThemeColorIntent.default,
+    disabled: false,
+    error: undefined,
+    errorStateControl: undefined,
+    flex: false,
+    max: 100,
+    min: 0,
+    step: 1,
+    value: '',
+    forbiddenValues: [],
+    formControl: undefined
+  };
+
+
+  async function setup({ attributes = {} }: { attributes?: Partial<OdsRangeAttribute> }) {
```

Comment:
you can add `{}` to avoid changing all the empty setup calls `{ attributes?: Partial<OdsRangeAttribute> } = {}`